meson.build

```diff
@@ -1874,7 +1874,7 @@ library_path_var = ''
 
 if host_machine.system() == 'windows'
   # nothing to do
-elif host_machine.system() == 'openbsd'
+elif host_machine.system() == 'openbsd' or host_machine.system() == 'netbsd'
```

Comment:
Hm? That wasn't necessary the last time I tried it on netbsd?

Reply:
I am not sure, it still fails: https://cirrus-ci.com/task/5865201915396096

Reply:
Done.

Reply:
As mentioned, this shouldn't be needed for netbsd - can you open an issue for trying to fix it?